README.md

```diff
@@ -71,14 +72,20 @@ cd /path/to/your/development/workspace
 git clone https://github.com/g3w-suite/g3w-admin.git --single-branch --branch dev ./g3w-admin
 ```
 
-Create a [**virtualenv**](https://g3w-suite.readthedocs.io/en/latest/install.html#create-virtualenv) from within your g3w-admin local repository:
+Create and activate [**virtualenv**](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) within your g3w-admin local repository:
 
 ```sh
 cd ./g3w-admin
+
+python3 -m venv --system-site-packages [--prompt g3w-admin] venv
 ```
 
 ```sh
-mkvirtualenv --system-site-packages .
+# Linux / Mac OS
+source venv/bin/activate
+
+# Windows
+.\venv\Scripts\activate
```

Comment:
Just to avoid making life difficult for newcomers/casual users by asking them to install any other additional software first.

BTW, nowadays IDEs are fully able to do all of that without any major pains, eg: [Using Python environments in VS Code](https://code.visualstudio.com/docs/python/environments#_creating-environments)

Reply:
OK